### ISSUE TITLE
GPU Relief: 'npm start' command line arg or flag request

### ISSUE THREAD

**Issue by michaelpalumbo** [worldmaking/alicelib_node#2]
Hi Graham, my laptop runs pretty hot whenever I'm running the server, and I suspect it is the alicenode video. Could we add a CLI argument or flag to the 'npm start' that either disables the video or changes the framerate (or something else that reduce the load on my computer's GPU?) Thanks!

**Comment by grrrwaaa** [comment]
Good point -- just added code to throttle the frame rate to around 30hz for now, to save our GPUs.

**Comment by grrrwaaa** [comment]
I don't think this should be a CLI arg, as there will be multiple windows before long. Rather I think we should have a config.json in the project folder that defines these things.

**Comment by michaelpalumbo** [comment]
Perfect. I'll keep the config.json in mind for git as well. Saw that you changed the framerate for the sim, thanks!

**Comment by michaelpalumbo** [comment]
Hi Graham,
Now that we have 'space' to pause the animation, is there any way we could add a keyboard command to pause the rendering?
